templates/openebs-operator-1.5.0.yaml

```diff
@@ -125,6 +125,14 @@ spec:
           value: "true"
         - name: OPENEBS_IO_INSTALL_DEFAULT_CSTOR_SPARSE_POOL
           value: "false"
+        - name: OPENEBS_IO_CSTOR_TARGET_DIR
+          value: "/var/openebs/sparse"
```

Comment:
is this sparse?

Reply:
OPENEBS_IO_CSTOR_TARGET_DIR can be used to specify the hostpath to be used for saving the shared content between the side cars of cstor volume pod.
The default path used is `/var/openebs/sparse`